src/pages/stdlib/{StdlibSection.slug}.js

```diff
@@ -10,6 +10,16 @@ import nickelLanguageDefinition from "../../prism/nickel";
 
 import ReactMarkdown from 'react-markdown';
 
+// Icon taken from gatsby-remark-autolink-headers at https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-autolink-headers/src/index.js
+const AnchorIcon = () => {
```

Comment:
It's not very important, but as a note, I think we have some react icon library that we can use with something like `<icon something>` (I don't remember the name of the markup/component)

Reply:
But I feel it's also important that whatever icon we use here is the same as the one used in the manual. So I'm not sure replacing the hardcoded icon (like it is in `gatsby-remark-autolink-headers`) is a good idea.